### ISSUE TITLE
SMAR-39 ⁃ Configure Mail Settings in Smart Sensor Dashboard

### ISSUE THREAD

**Issue by thatdoogieguy** [thatdoogieguy/SST#151]
[~indr4_winata] configure the mail settings based on SMAR-38

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~admin]:
Can I get the information of:

  * system_sender_email (before: info@mapmybins.com.au)
  * SMTP host name (before: mail.bottletopdigital.com)
  * SMTP port
  * SMTP username
  * SMTP password
  * SMTP Crypto [ssl / tls]. (before: tls)

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~admin]: just a reminder for this one. You haven't give me a detail for this.

**Comment by thatdoogieguy** [comment]
➤ User  commented:

mail admin@smartsensor.com.au
pass 24Ld69JZgP

SMTP: mail.smartsensor.com.au
port: 587

Should still be TLS / STARTTLS

**Comment by thatdoogieguy** [comment]
➤ User  commented:

Google Details

Username
admin@smartsensor.com.au

Password
ud3EmQ*>

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~admin]:
Sorry, I tried to send email many times with correct smtp info, but it seems failed. And then, I tried to open the email using gmail.com, and it displayed "Acceptance" page, as well as some other information, like admin@smartsensor.com.au don't have access to change what google service etc. And at the end of that, I need to setup a new password. Can't use the same password.
So, I change the password to:

Password: ud3EmQ*>i

and then the smtp is working
